### ISSUE TITLE
Does not work with latest Storybook beta release.

### ISSUE THREAD

**Issue by joepuzzo** [tuchk4/storybook-readme#79]
I cant seem to get this to work with the latest storybook beta release. Im in the process of trying to track down if the addon api changed or something but its currently broken.

**Comment by joepuzzo** [comment]
Maybe if you could just help me learn how this works i can debug.. i have looked trough every file in your source code and i see that the way the README gets injected is the following 

```
<div
              key={index}
              className={'markdown-body'}
              dangerouslySetInnerHTML={{ __html: doc }}
            />
```

But i dont understand how that turns markdown into HTML?? Am i missing something. There must be some function that you call to translate the MD to MarkUp haha?

**Comment by joepuzzo** [comment]
Ok i traked it down to the new version of storybook ( Beta ) Not using the html and markdown loader.. now my only issue left is the classes are not getting added to the parsed html tags. Any ideas?

**Comment by joepuzzo** [comment]
Ok the classes are getting added but im getting a bunch of `\n` showing up.. its prob somthing stupid

**Comment by joepuzzo** [comment]
Ok so for those that run into this same issue.. the solve was to manually override the storybookBase config, in a custom webpack config under the `.storybook` directory. This is by no means a good solve but it looks like the beta release just uses raw loader where it used to use html and markdown loader. 
```
storybookBaseConfig.module.rules[1] = {
    test: /\.md$/,
    use: [
      {
        loader: 'html-loader',
      },
      {
        loader: 'markdown-loader',
        options: {
        }
      },
    ],
    include: path.resolve(__dirname, "../"),
    exclude: /node_modules/
  };
```

**Comment by natterstefan** [comment]
Hi @joepuzzo,

thanks for the insights. Definitly helps, but have you experience this issue as well:

![image](https://user-images.githubusercontent.com/1043668/43333904-16c057e8-91cc-11e8-8030-b1f7aafd5a99.png)

My webpack.config.js looks like this currently

```js
```

**Comment by joepuzzo** [comment]
Yea i have! Im not at my comp rigt now and forget the solve but i will look later

**Comment by joepuzzo** [comment]
Or rather u can go check out my informed lib on my github because thats configured correctly

**Comment by tuchk4** [comment]
Sorry guys, I am on vacation and without my mac. Will get back at monday
and help to resolve your issues.

**Comment by natterstefan** [comment]
thank you @joepuzzo, I am going to check [it](https://github.com/joepuzzo/informed/blob/master/.storybook/webpack.config.js) out.

Enjoy your vacation @tuchk4  :)

**Comment by natterstefan** [comment]
Hi @tuchk4, did you find some time to investigate this issue further?

**Comment by aloneszjl** [comment]
same issue

**Comment by tuchk4** [comment]
@natterstefan going to investigate this issue today

**Comment by natterstefan** [comment]
Thank you very much @tuchk4, sounds great!

**Comment by tuchk4** [comment]
@morficus  going to publish new version today.

**Comment by tuchk4** [comment]
Sorry for the delay

Published `4.0.0-beta1`. Can you please take a look and ping if any problems.

**Comment by natterstefan** [comment]
Hi @tuchk4, it works for me. Thank you!

**Comment by morficus** [comment]
Yup! the 4.0.0-beta1 works for me as well!
Thank you very much!

**Comment by tuchk4** [comment]
released in `4.0.0`

**Comment by natterstefan** [comment]
@tuchk4 awesome, thanks you very much!
